introduction.md

```diff
@@ -0,0 +1 @@
+Hello, my name is Timothy Barnes. I am a mentor for Bloc.io. My shoes are off.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.